build/dt-report-generator-bundle.js

```diff
@@ -7,13 +7,38 @@
 
 const browserify = require('browserify');
 const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
 
-const distDir = __dirname + '/../dist';
-const outFile = `${distDir}/report-generator.js`;
+const distDir = path.join(__dirname, '..', 'dist', 'dt-resources');
+const bundleOutFile = `${distDir}/report-generator.js`;
 const generatorFilename = `./lighthouse-core/report/report-generator.js`;
-browserify(generatorFilename, {standalone: 'ReportGenerator'})
-  .transform('brfs')
+const htmlReportAssets = require('../lighthouse-core/report/html/html-report-assets.js');
+
+/**
+ * Used to save cached resources (Runtime.cachedResources). Content must be converted to ascii.
+ * @param {string} name
+ * @param {string} content
+ */
+function convertToAsciiAndWriteFile(name, content) {
+  // eslint-disable-next-line no-control-regex
+  const escaped = content.replace(/[^\x00-\x7F]/g, c => '\\\\u' + c.charCodeAt(0).toString(16));
+  fs.writeFileSync(`${distDir}/${name}`, escaped);
+}
+
+rimraf.sync(distDir);
+fs.mkdirSync(distDir);
+
+convertToAsciiAndWriteFile('report.js', htmlReportAssets.REPORT_JAVASCRIPT);
+convertToAsciiAndWriteFile('report.css', htmlReportAssets.REPORT_CSS);
+convertToAsciiAndWriteFile('template.html', htmlReportAssets.REPORT_TEMPLATE);
+convertToAsciiAndWriteFile('templates.html', htmlReportAssets.REPORT_TEMPLATES);
```

Comment:
asserting something about `htmlReportAssets` in here might catch at least some issues whenever `yarn build-all` is run. Could be something stupid simple like `Object.keys` is only the ones we expect. WDYT?

Reply:
how's that?

Reply:
> how's that?

seems good?